macros/edr/tests/on_run_end/handle_tests_results.sql

```diff
@@ -100,9 +100,16 @@
 {% macro get_metric_test_result(run_result_dict, anomaly_dict, test_node, test_anomaly_scores_table) %}
     {% set full_table_name = elementary.insensitive_get_dict_value(anomaly_dict, 'full_table_name', '') %}
     {% set split_full_table_name = full_table_name.split('.') %}
-    {% set database_name = split_full_table_name[0] %}
-    {% set schema_name = split_full_table_name[1] %}
-    {% set table_name = split_full_table_name[2] %}
+    {# Databricks full name is schema.table, no db #}
```

Comment:
This feels like it should be an adapter macro 🤔 
What if another WH won't support schemas? Then we will need to add another if statement which implies that an adapter macro will fit good here IMO

Reply:
The thing is adapter would limit us to config per integration, and Databricks might have a database name.